numpyro/infer/hmc.py

```diff
@@ -194,6 +194,7 @@ def init_kernel(
         max_tree_depth=10,
         find_heuristic_step_size=False,
         forward_mode_differentiation=False,
+        regularize_mass_matrix=True,
```

Comment:
nit: consider clearly separating positional from kwarg-only arguments everywhere, e.g.
```diff
  def init_kernel(
      init_params,
      num_warmup,
+     *,
      step_size=1.0,
      ...
```
This ensures backwards compatibility when you insert a new kwarg like `regularize_mass_matrix` in the middle.